model/fn-execution/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
```

Comment:
<!--new_thread; commit:f0b52c35fe2ad9680b8b41a35f95d8acb800da03; resolved:0-->
It seems like we are not required to blanket apply the root `build_rules.gradle`. It might be cleaner to break it up into things that really are across all modules and separately things that are not. For example, checkstyle, findbugs, etc should not even by loaded for building Python, Go, etc. So those configurations would go in a root level `java_base.gradle` or some such.